src/tracker-component.js

```diff
@@ -66,15 +66,32 @@ type Config = {|
     paramsToBeaconUrl? : ParamsToBeaconUrl
 |};
 
+const getUserId = () => {
+    return document.cookie ? document.cookie : localStorage.getItem('user-id');
+};
+
+const setRandomUserId = () => {
+    localStorage.setItem('user-id', generateId());
```

Comment:
Same here.

Reply:
here I'm setting the user id in localStorage, is that okay?